src/devices/custom_devices_diy.ts

```diff
@@ -1305,6 +1305,24 @@ const definitions: Definition[] = [
             await endpoint.read('genBasic', ['modelId', 'swBuildId', 'powerSource']);
         },
     },
+    {
+        zigbeeModel: ['ptvo_counter_2ch'],
+        model: 'ptvo_counter_2ch',
+        vendor: 'Custom devices (DiY)',
+        description: '[Configurable firmware](https://ptvo.info/zigbee-configurable-firmware-features/)',
```

Comment:
Could you also create some instructions on how to create this device?

Reply:
> Could you also create some instructions on how to create this device?

Hi!
All can be found in this article from author :
https://modkam.ru/2021/04/29/schetchik-impulsov-zigbee/

Reply:
It's in Russian and therefore not understandable for a large audience, do you have an English version? If not, I propose to keep this as an ext converter.

Reply:
OK, I will translate it and post link

Reply:
> It's in Russian and therefore not understandable for a large audience, do you have an English version? If not, I propose to keep this as an ext converter.

https://dzen.ru/a/ZUnKTY3_Dkue45gb